client/css/style.css

```diff
@@ -18,7 +36,26 @@ html, body, #main, #app {
 }
 
 body {
-    background: #f1f2f7;
+    font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+    font-size: 13px;
+    color: #717171;
+    background-color: #f1f2f7;
+    -webkit-font-smoothing: antialiased;
```

Comment:
Can you please remove this non-standard property? (see also https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth)